nnstreamer/tensor_filter/meson.build

```diff
@@ -26,5 +26,5 @@ static_library('nnstreamer_filter_nntrainer',
   dependencies: nnstreamer_filter_nntrainer_deps,
   include_directories: nntrainer_inc,
   install: true,
-  install_dir: nnstreamer_libdir
+  install_dir: filter_subplugin_install_dir
```

Comment:
I thought this should be /usr/lib/. Not /usr/lib/nnstreamer/filtesr/

Reply:
this is the nnstreamer filter extension, so install in /usr/lib/nnstreamer/filters?
in /usr/lib, we have libnntrainer.so corresponding to the libnnstreamer.so

Reply:
my bad, updated.